code/app/src/main/java/com/example/the_tarlords/data/QR/QRCode.java

```diff
@@ -231,7 +231,7 @@ public void findPastEvents(String user,  EventsCallback callback) {
      * @return           True if the event has passed; otherwise, false
      */
     public boolean pastEvent(String startDate, String endTime) {
-        DateTimeFormatter formatter1 = DateTimeFormatter.ofPattern("MMMM d, yyyy", Locale.ENGLISH);
+        DateTimeFormatter formatter1 = DateTimeFormatter.ofPattern("MMM dd, yyyy", Locale.ENGLISH);
```

Comment:
Month has 3 M's ?

Reply:
Jan, Feb etc, I reformatted the date to use the first three letters instead of the whole word, seemed to work when I was trying it yesterday